instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -21,18 +22,10 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         compatible do
-          # ActiveSupport::Notifications were introduced in Grape 0.13.0
-          # https://github.com/ruby-grape/grape/blob/master/CHANGELOG.md#0130-2015810
           gem_version >= MINIMUM_VERSION
         end
 
-        # Config options available in ddog Grape instrumentation
-        option :enabled, default: true, validate: :boolean
```

Comment:
This option is available for all instrumentations (as I explained in the README, in the Configuration options section).